src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -292,6 +393,31 @@ private string ClearHexString(string hexString)
             }
         }
 
+        /// <summary>
+        /// Converts an address to its corresponding scripthash (little-endian)
+        /// </summary>
+        /// <param name="address">
+        /// String that represents the address to be converted
+        /// </param>
+        /// <returns>
+        /// Returns null when the string does not represent an address or when
+        /// it is not possible to parse the address to scripthash; otherwise returns
+        /// the string that represents the converted scripthash
+        /// </returns>
+        private string? AddressToScripthashLE(string address)
+        {
+            try
+            {
+                var bigEndScript = address.ToScriptHash(NeoSystem.Settings.AddressVersion);
+
+                return bigEndScript.ToArray().ToHexString();
+            }
+            catch
+            {
+                return null;
+            }
+        }
```

Comment:
Could you please add the following compatibility test for this API: convert address `NRQJbjKWgcLX214Kbx6PDH2CHJdqbsSi4Q` to LE script hash. I'd like to ensure that NeoGo has the same LE/BE conversion result:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ ./bin/neo-go util convert NRQJbjKWgcLX214Kbx6PDH2CHJdqbsSi4Q
Address to BE ScriptHash	3c347b368199840085f7d92704a14becd578fa3d
Address to LE ScriptHash	3dfa78d5ec4ba10427d9f78500849981367b343c
Address to Base64 (BE)		PDR7NoGZhACF99knBKFL7NV4+j0=
Address to Base64 (LE)		Pfp41exLoQQn2feFAISZgTZ7NDw=
String to Hex				4e52514a626a4b5767634c583231344b6278365044483243484a6471627353693451
String to Base64			TlJRSmJqS1dnY0xYMjE0S2J4NlBESDJDSEpkcWJzU2k0UQ==
```

Reply:
```csharp
public static UInt160? TryParseUInt160(string? value)
{
    if (string.IsNullOrEmpty(value)) return default;
    if (UInt160.TryParse(value, out var result))
        return result;
    return UInt160.Zero;
}
```